src/plugins/vis_type_timeseries/common/fields_utils.ts

```diff
@@ -6,17 +6,55 @@
  * Side Public License, v 1.
  */
 
+import { i18n } from '@kbn/i18n';
 import { FieldSpec } from '../../data/common';
 import { isNestedField } from '../../data/common';
-import { SanitizedFieldType } from './types';
+import { FetchedIndexPattern, SanitizedFieldType } from './types';
 
-export const toSanitizedFieldType = (fields: FieldSpec[]) => {
-  return fields
-    .filter(
-      (field) =>
-        // Make sure to only include mapped fields, e.g. no index pattern runtime fields
-        !field.runtimeField && field.aggregatable && !isNestedField(field)
-    )
+export class FieldNotFoundError extends Error {
+  constructor(name: string) {
+    super(
+      i18n.translate('visTypeTimeseries.fields.fieldNotFound', {
```

Comment:
If there are more than two runtime fields, and I remove them both, then TSVB is broken
![image](https://user-images.githubusercontent.com/17003240/114379623-b38f2c80-9b91-11eb-9d7e-503f75e6cef7.png)